src/Bridge/Doctrine/EventListener/ManagerViewListener.php

```diff
@@ -65,8 +66,11 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
                 $event->setControllerResult(null);
             break;
         }
-
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
Yes it is better to catch this one. Can you finish this PR? Thanks for working on this!